local-cli/desktop/genDesktop.js

```diff
@@ -24,19 +24,19 @@ function applyDesktopPlatformPatch() {
    });
 }
 
-function applyIncreseBundlerMemoryPatch() {
-  exec('pwd && patch --verbose -d ./node_modules/metro/src/JSTransformer -i ../../../react-native/increase_bundler_memory_usage.patch', (err, stdout, stderr) => {
-     console.log(`Std output: ${stdout}`);
-     if (err) {
-       console.error(`exec error: ${err}`);
-       return;
-     }
-   });
-}
+// function applyIncreseBundlerMemoryPatch() {
+//   exec('pwd && patch --verbose -d ./node_modules/metro/src/JSTransformer -i ../../../react-native/increase_bundler_memory_usage.patch', (err, stdout, stderr) => {
+//      console.log(`Std output: ${stdout}`);
+//      if (err) {
+//        console.error(`exec error: ${err}`);
+//        return;
+//      }
+//    });
+// }
 
 function genDesktop(args, config) {
   applyDesktopPlatformPatch();
-  applyIncreseBundlerMemoryPatch();
+  // applyIncreseBundlerMemoryPatch();
```

Comment:
@MaxRis - Is this patch still needed with the latest metro bundler?  If so, can you please give an updated patch?

Reply:
@madhavarshney I believe so. metro bundler fails with out of memory on JS heap when running on too long JS input bundles (more than few MBs). We will know for sure if will try to build status-react project against your PR. It will fail if the patch still needed.

Reply:
@MaxRis - I think this issue is a fairly specific use case and `react-native-desktop` shouldn't be the one to patch metro.  Projects that need this patch should implement it in their own repository, as @vkjr [has done here](https://github.com/status-im/status-react/pull/6983/commits/de36dc63319ee0e04c88727f711921913b8db260).  We can document this in the troubleshooting page.

Reply:
@madhavarshney makes sense!